includes/class-sensei-course.php

```diff
@@ -2189,7 +2194,8 @@ public function the_course_meta( $course_id ) {
 		do_action( 'sensei_course_meta_inside_before', $course->ID );
 
 		echo '<span class="course-lesson-count">' .
-			esc_html( Sensei()->course->course_lesson_count( $course->ID ) ) . '&nbsp;' . esc_html__( 'Lessons', 'sensei-lms' ) .
+			// translators: Placeholder %d is the lesson count.
+			sprintf( esc_html( _n( '%d Lesson', '%d Lessons', $lesson_count, 'sensei-lms' ) ), intval( $lesson_count ) ) .
```

Comment:
Again, we don't need the `intval` here as it's already guaranteed to be an integer.